calico-cloud_versioned_docs/version-18/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,51 +2,52 @@
-    - For each CVE listed, you can view the details of the vulnerability or add an exception to ignore the vulnerability (see [Create vulnerability exceptions](#create-vulnerability-exceptions)). 
+## Running Images tab
 
-**Runtime tab**
+The tab lists active container images *for all connected managed clusters*. It provides the CVEs associated with running pods to help you assess pod vulnerability. If you are using your own scanner, you can export running image URLs to your backend system. This tab is disabled by default.
 
-The left panel lists images running in your cluster. Because you can see the CVEs associated with running pods, you can assess pod vulnerability. The columns, **Clusters** and **Running Instances**, show the number of running instances in clusters that are connected to {{prodname}}.
-
-The right panel lists:
-
-- Number of running pods that have images with scan results of Fail, Warn, Pass, and Unknown (the CLI scanner has not yet reported results)
-- High Risk Namespaces (top five namespaces based on total Fails)
-- Runtime details (when you have an image selected from the left panel list) shows the clusters and namespaces the selected image is running in
+To enable Running Images, click the **Setting** icon in the top right corner, then select **Enable Runtime View**.
 
 :::note
 
-In the Scan Results and Runtime tabs, the **Registry path** field may be blank if {{prodname}} cannot access this metadata. For example, images from Docker Hub do not specify the registry in the image metadata.
+If you are using the CLI scanner and your cluster does not use the default containerd socket path (`/run/containerd/containerd.sock`), you must change the path to allow the Running Images service to collect image information. To update the CRI socket path for a cluster, run the following command:
+
+```bash
+kubectl patch imageassurance default --type='merge' -p '{"spec":{"criSocketPath":"<path-to-cri-socket>"}}'
+```
+For details, see [Image Assurance installation reference](../reference/installation/api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec).
 
 :::
 
-**How Pass, Warn, and Fail scores map to CVSS scores**
+Other notes:
 
-{{prodname}} image scoring is based on the Common Vulnerability Scoring System v3 (CVSS Scores).
-You can change the CVSS score threshold between a Pass and a Warn, and between a Warn and a Fail. Currently, only [CVSS v3 scores](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) are displayed. CVEs that do not have a CVSS v3 score (too old to have one, or are too new to be assigned one), display a blank score in the UI, and **N/A** in the CLI.
+- The columns, **Clusters** and **Running Instances**, show the number of running instances in clusters that are connected to {{prodname}}.
+- The **Unknown** scan result filter is disabled by default. To enable it, click the **Result** drop-down menu and select **Unknown**.
```

Comment:
Is there any place in our docs that explains what an "Unknown" scan result actually means (i.e. that the image as not been fully scanned yet)? 

It might be worth some elaboration for the user.

Reply:
Added a blurb.